src/raftstore/store/peer.rs

```diff
@@ -966,6 +1047,19 @@ impl Peer {
                 // apprently, all the callbacks whose term is less than entry's term are stale.
                 self.notify_not_leader(cmd);
             }
+            // Update the leader lease when the leader commits the first empty entry
+            // during its term.
+            if self.is_leader() && term == self.raft_group.raft.term {
+                let now = clocktime::now_monotonic_raw_clocktime();
+                let next_expired_time =
```

Comment:
L530 updates the `leader_lease_expired_time` to `become_leader_time + election_timeout`; Here updates it again to `now + election_timeout - (now - become_leader_time)`, which is exactly `become_leader_time + election_timeout`. So what's point to update here? And it's not guaranteed that all empty entries are the first entry at their term.